newbank/server/NewBank.java

```diff
@@ -1,17 +1,20 @@
 package newbank.server;
 
 import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 
 public class NewBank {
 
   private static final NewBank bank = new NewBank();
   private HashMap<String, Customer> customers;
+  private ArrayList<String> commands = new ArrayList<String>();
 
   private NewBank() {
     customers = new HashMap<>();
     addTestData();
+    addCommands;

```

Comment:
needs to be addCommands(commands);

Reply:
Lots of silly mistakes- you can tell I'm getting a bit frustrated with the process.